check-diff.sh

```diff
@@ -245,7 +252,7 @@ function set_environment_variables {
 
 	cat "$TEMP_DIRECTORY/paths-scope" | grep -E '\.php(:|$)' | cat - > "$TEMP_DIRECTORY/paths-scope-php"
 	cat "$TEMP_DIRECTORY/paths-scope" | grep -E '\.jsx?(:|$)' | cat - > "$TEMP_DIRECTORY/paths-scope-js"
-	cat "$TEMP_DIRECTORY/paths-scope" | grep -E '\.(css|scss)(:|$)' | cat - > "$TEMP_DIRECTORY/paths-scope-scss"
+	cat "$TEMP_DIRECTORY/paths-scope" | grep -E '\.(css|scss)(:|$)' | cat - > "$TEMP_DIRECTORY/paths-scope-css"
```

Comment:
Let's drop `scss` from the above and only go with `css` initially.

The `stylelint-config-wordpress` configuration includes two _stylelint shared configs_, one for CSS and one for SCSS, so let's get CSS up and running and follow up with with SCSS in another PR